php/templates/deployment.yaml

```diff
@@ -190,6 +190,13 @@ spec:
           mountPath: /usr/local/etc/php-fpm.conf
         - name: fpm-confd
           mountPath: /usr/local/etc/php-fpm.d
+        - name: fpm-php-ini
+          subPath: php.ini
+          mountPath: /usr/local/etc/php/php.ini
+{{- if .Values.newrelic.enabled }}
+        - name: newrelic
+          mountPath: /tmp
+{{- end }}
```

Comment:
You can add volume mounts with using `fpm.extraVolumeMounts`.  I think that doesn't need to add the code.

Reply:
Is the comment appropriate? Also I misunderstood?